doc/source/news.rst

```diff
@@ -5,6 +5,408 @@
+     #             "Mroonga"
+     #           ],
+     #           2
+     #         ],
+     #         [
+     #           [
+     #             "Groonga",
+     #             "PGroonga"
+     #           ],
+     #           2
+     #         ],
+     #         [
+     #           [
+     #             "Mroonga",
+     #             "PGroonga"
+     #           ],
+     #           1
+     #         ],
+     #         [
+     #           [
+     #             "Groonga",
+     #             "Mroonga",
+     #             "PGroonga"
+     #           ],
+     #           1
+     #         ]
+     #       ]
+     #     }
+     #   ]
+     # ]
+
+  This result shows following.
+  
+  .. csv-table::
+
+     "tag","number of occurrence"
+     "``Groonga``", "4"
+     "``Mroonga``", "2"
+     "``PGroonga``", "2"
+     "``Groonga`` and ``Mroonga``", "2"
+     "``Groonga`` and ``PGroonga``", "2"
+     "``Mroonga`` and ``PGroonga``", "1"
+     "``Groonga`` and ``Mroonga`` and ``PGroonga``", "1"
+
+  This feature is complex. For more information, please refer to  :ref:`select-drilldowns-label-key-vector-expansions-power-set`.
+
+* [:doc:`reference/commands/select`] Specific element of vector column is now able to be search target. 
+
+  It allows specific elements of vector column to be search targets that specifying the specific elements to ``match_columns`` with index number. 
+
+  Following is a sample case.
+  
+  .. code-block::
+
+     table_create Memos TABLE_NO_KEY
+     column_create Memos contents COLUMN_VECTOR ShortText
+
+     table_create Lexicon TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
+     column_create Lexicon memo_index COLUMN_INDEX|WITH_POSITION|WITH_SECTION Memos contents
+     load --table Memos
+     [
+     ["contents"],
+     [["I like Groonga", "Use Groonga with Ruby"]],
+     [["I like Ruby", "Use Groonga"]]
+     ]
+     select Memos \
+       --match_columns "contents[1]" \
+       --query Ruby \
+       --output_columns "contents, _score"
+     # [
+     #   [
+     #     0,
+     #     0.0,
+     #     0.0
+     #   ],
+     #   [
+     #     [
+     #       [
+     #         1
+     #       ],
+     #       [
+     #         [
+     #           "contents",
+     #           "ShortText"
+     #         ],
+     #         [
+     #           "_score",
+     #           "Int32"
+     #         ]
+     #       ],
+     #       [
+     #         [
+     #           "I like Groonga",
+     #           "Use Groonga with Ruby"
+     #         ],
+     #         1
+     #       ]
+     #     ]
+     #   ]
+     # ]
+
+  ``--match_columns "contents[1]"`` specifies only 2nd vector elements of ``contents`` as the search target. 
+  In this sample, ``["I like Groonga", "Use Groonga with Ruby"]`` is shown in the results because 2nd element ``Ruby`` is in ``Use Groonga with Ruby``. However, ``["I like Ruby", "Use Groonga"]`` is not shown in results because 2nd element ``Ruby`` is not in ``Use Groonga``.
```

Comment:
```2nd element ``Ruby`` is in ``Use Groonga with Ruby`` ``` の部分が構文解析できないのですが、以下のような感じではどうでしょうか？

```suggestion
  In this sample, ``["I like Groonga", "Use Groonga with Ruby"]`` is shown in the results because ``Ruby`` is in 2nd element ``Use Groonga with Ruby``. However, ``["I like Ruby", "Use Groonga"]`` is not shown in results because 2nd element ``Ruby`` is not in ``Use Groonga``.
```

Reply:
すみません、一旦上記の変更を入れてマージしてしまいました。
もしこれではおかしい、ということであればご指摘頂ければ後ほど反映します。

Reply:
遅くなってすみません。
2nd element = ``Use Groonga with Ruby``
であればOKです
が私の認識では
 2nd element＝ ``Ruby``　だったので、最初の語順になってました。
2要素目は``Ruby``で、``Ruby``が、``Use Groonga``に入っていないから結果に表示されないという理解でしたがちがいましたか？

Reply:
2nd element = `Use Groonga with Ruby` の意図でした！

`["I like Groonga", "Use Groonga with Ruby"]` の第一要素が `I like Groonga` で、第二要素が `Use Groonga with Ruby` という感じです。
`["I like Ruby", "Use Groonga"]` についても同様に 第一要素が `I like Ruby"`、第二要素が `Use Groonga"` です。

`Ruby` という単語が、 `["I like Groonga", "Use Groonga with Ruby"]` の第二要素には入っているけれども、 `["I like Ruby", "Use Groonga"]` の第二要素には入っていないので、 `["I like Ruby", "Use Groonga"]` は結果には表示されない、という感じです！

Reply:
それであれば大丈夫です。
element　という事だったので「1ワード」かなと思ってしましました。
難しいですね・・・